scenarios/06_Elensefar.cfg

```diff
@@ -56,9 +56,9 @@
         side=2
         controller=ai
         defeat_condition=no_units_left
-        recruit=Cavalryman, Dragoon, Horseman, Knight, Spearman, Swordsman, Bowman, Longbowman
+        recruit=Cavalryman, Dragoon, Horseman, Knight, Spearman, Swordsman, Bowman, Longbowman, Heavy Infantryman, Shock Trooper
         {GOLD 200 400 600}
-        {INCOME 15 30 45}
+        village_gold=4
```

Comment:
Interesting. This never occurred to me. Looks like it should work well.

Reply:
This kind of income bonus does seem to help an AI driven ally be more impactful, as the AI is especially good at village capture. I should give it a try on _**02_The_Raid**_ as well.